resources/lang/en/thoughts/alive/apprentice.json

```
@@ -840,7 +840,7 @@
             "Wishes r_c could be a bit more understanding",
             "Feels like r_c doesn't like {PRONOUN/m_c/object}",
             "Is surprised when r_c gives {PRONOUN/m_c/object} the day off on a hard day",
-            "Gets a lecture on being more confident when {PRONOUN/m_c/subject} {VERB/m_c/complain/complains} about never succeeding with a particular move",
+            "Was lectured on being more confident when {PRONOUN/m_c/subject} {VERB/m_c/complain/complains} about never succeeding with a particular move",
```

```suggestion
            "Complained about never succeeding with a particular move and was lectured on being more confident",
```
imo this is a smoother wording and a tad shorter